client/src/i18n/locales/es/index.json

```diff
@@ -30,7 +30,8 @@
   "multiBusinessPlaceholder": "Elija uno",
   "multiBusinessFalse": "No, solo gestiono 1 negocio de cuidado infantil",
   "multiBusinessTrue": "Sí, gestiono varios negocios de cuidado infantil",
-  "phone": "Número de teléfono (solo llamaremos o enviaremos un mensaje de texto si lo deseas)",
+  "phone": "Número de teléfono",
+  "phoneNote": "solo llamaremos o enviaremos un mensaje de texto si lo deseas",
   "phoneTypePlaceholder": "Elija uno",
```

Comment:
@arku I know this is outside this particular pull request, but can you also change this - we missed it last time:

"gettingStartedDetailsTitle": "Agrega la información de su empresa" --> "Agrega la información de tu empresa"

Reply:
Done.